sample_db/jsonschemas/sampledb/properties.json

```diff
@@ -0,0 +1,130 @@
+            "enum": [
+                "TerraCollect",
+                "QGIS",
+                "CollectEarth"
+            ]
+        },
+    	"collections_type": {
+            "type": "string",
+            "title": "The collection identifier.",
+            "description": "The collection identifier used in Brazil Data Cube STAC."
+        },
+        "stac_cloud_cover": {
+            "type": "number",
+            "title": "The stac cloud cover factor.",
+            "description": "The stac cloud cover factor.",
+            "default": 80,
+            "minimum": 0,
+            "maximum": 100,
+            "examples": [
+                "0.9"
+            ]
+        },
+        "external_type": {
+            "type": "object",
+             "required": [
+                "collection",
+                "rgb",
+                "url"
+            ],
+            "properties": {
+                "collection": {
+                    "type": "string",
+                    "title": "The collection map name.",
+                    "description": "The collection map name.",
+                    "examples": [
+                        "Planet Maps"
+                    ]
+                },
+                "url": {
+                    "type": "string",
+                    "title": "The collection map url.",
+                    "description": "The collection map url.",
+                    "examples": [
+                        "https://earth-search.aws.element84.com/v0/"
+                    ]
+                },
+                "rgb": {
+                    "type": "object",
+                     "required": [
+                        "red",
+                        "blue",
+                        "green"
+                    ],
+                    "properties": {
+                        "red": {
+                            "type": "string",
+                            "title": "The red band.",
+                            "description": "The red band.",
+                            "examples": [
+                                "B04"
+                            ]
+                        },
+                        "blue": {
+                            "type": "string",
+                            "title": "The blue band.",
+                            "description": "The blue band.",
+                            "examples": [
+                                "B02"
+                            ]
+                        },
+                        "green": {
+                            "type": "string",
+                            "title": "The green band.",
+                            "description": "The green band.",
+                            "examples": [
+                                "B03"
+                            ]
+                        }
+                    }
+                }
+            }
+        }
+
+    },
+    "additionalProperties": false,
+	"properties": {
+		"name-type": {
```

Comment:
This JSON schema seems invalid related ``name-type`` reference. Could you check? i have tested in [https://www.jsonschemavalidator.net/](https://www.jsonschemavalidator.net/).

I have noticed that there are properties in the field ``required`` that do not exist.